crates/nu-command/src/conversions/into/binary.rs

```diff
@@ -82,6 +89,22 @@ impl Command for IntoBinary {
                     Span::test_data(),
                 )),
             },
+            Example {
+                description: "convert a number to a nushell binary primitive (big endian)",
+                example: "258 | into binary --endian big",
+                result: Some(Value::binary(
+                    i64::from(258).to_be_bytes().to_vec(),
+                    Span::test_data(),
+                )),
+            },
+            Example {
+                description: "convert a number to a nushell binary primitive (little endian)",
+                example: "258 | into binary --endian little",
+                result: Some(Value::binary(
+                    i64::from(258).to_le_bytes().to_vec(),
+                    Span::test_data(),
+                )),
+            },
```

Comment:
Debatable point: I personally think with how examples are layed out in the `help` it is often economical to only show one kind of use of a particular flag. (That we have testing for them, shouldn't serve as *the* way to test the behavior anyways but rather serves as a check to keep documentation in line with a coverage bonus.)

At the same time `into binary` isn't too crowded at the moment so is fine (and maybe serves as a helpful reminder for what that endianness nonsense actually means